Fix deprecated system colors to match same-as requirements in CSS Color 4

obyknovenius opened the pull request:
#### 29dde143ed81fa034f7d5f1fd59e78139f0670e1
<pre>
Fix deprecated system colors to match same-as requirements in CSS Color 4
<a href="https://bugs.webkit.org/show_bug.cgi?id=245609">https://bugs.webkit.org/show_bug.cgi?id=245609</a>

Reviewed by Aditya Keerthi.

CSS Color 4 deprecates some of the system colors
and adds a &quot;same-as&quot; requirement for each of them.

<a href="https://drafts.csswg.org/css-color-4/#deprecated-system-colors">https://drafts.csswg.org/css-color-4/#deprecated-system-colors</a>

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/system-color-compute-expected.txt:
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/css/css-color/system-color-compute-expected.txt: Removed.
* LayoutTests/platform/glib/tables/mozilla/bugs/bug103533-expected.txt:
* LayoutTests/platform/gtk/TestExpectations:
* LayoutTests/platform/ios-wk2/TestExpectations:
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/css/css-color/system-color-compute-expected.txt: Removed.
* LayoutTests/platform/ios/tables/mozilla/bugs/bug103533-expected.txt:
* LayoutTests/platform/mac/TestExpectations:
* LayoutTests/platform/mac/imported/w3c/web-platform-tests/css/css-color/system-color-compute-expected.txt: Copied from LayoutTests/imported/w3c/web-platform-tests/css/css-color/system-color-compute-expected.txt.
* LayoutTests/platform/mac/tables/mozilla/bugs/bug103533-expected.txt:
* Source/WebCore/rendering/RenderTheme.cpp:
(WebCore::RenderTheme::systemColor const):
* Source/WebCore/rendering/RenderThemeAdwaita.cpp:
(WebCore::RenderThemeAdwaita::systemColor const):
* Source/WebCore/rendering/RenderThemeIOS.mm:
(WebCore::RenderThemeIOS::systemColor const):
* Source/WebCore/rendering/RenderThemeMac.mm:
(WebCore::RenderThemeMac::systemColor const):

Originally-landed-as: 268124@main (a396d027d051). <a href="https://bugs.webkit.org/show_bug.cgi?id=245609">https://bugs.webkit.org/show_bug.cgi?id=245609</a>
Canonical link: <a href="https://commits.webkit.org/269740@main">https://commits.webkit.org/269740@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/4724348f3f814f095357f30efa60b9758a2b5958

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/23434 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/1547 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/24558 "Built successfully") | [⏳ 🛠 wpe ](https://ews-build.webkit.org/#/builders/WPE-Build-EWS "Waiting in queue, processing has not started yet") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/21643 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/23704 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/3094 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/16/builds/23984 "Built successfully") | [⏳ 🧪 wpe-wk2 ](https://ews-build.webkit.org/#/builders/WPE-Build-EWS "Waiting in queue, processing has not started yet") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/23675 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/1100 "Passed tests") | [  ~~🧪 api-mac~~](https://ews-build.webkit.org/#/builders/18/builds/20276 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/26197 "Built successfully") | 
| | [  ~~🧪 ios-wk2-wpt~~](https://ews-build.webkit.org/#/builders/42/builds/901 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/21168 "Passed tests") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/1/builds/27315 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/21363 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/21430 "Passed tests") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/21/builds/25183 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/893 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/17/builds/18618 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/856 "Built successfully") | | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/5594 "Built successfully and passed tests") | [  ~~🛠 watch~~](https://ews-build.webkit.org/#/builders/43/builds/1305 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/1159 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

pxlcoder reviewed (commented):
Can you explain what this does differently compared to the original change (268124@main) in the commit description?

obyknovenius committed:
Fix deprecated system colors to match same-as requirements in CSS Color 4
https://bugs.webkit.org/show_bug.cgi?id=245609

Reviewed by Aditya Keerthi.

CSS Color 4 deprecates some of the system colors
and adds a "same-as" requirement for each of them.

https://drafts.csswg.org/css-color-4/#deprecated-system-colors

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/system-color-compute-expected.txt:
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/css/css-color/system-color-compute-expected.txt: Removed.
* LayoutTests/platform/glib/tables/mozilla/bugs/bug103533-expected.txt:
* LayoutTests/platform/gtk/TestExpectations:
* LayoutTests/platform/ios-wk2/TestExpectations:
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/css/css-color/system-color-compute-expected.txt: Removed.
* LayoutTests/platform/ios/tables/mozilla/bugs/bug103533-expected.txt:
* LayoutTests/platform/mac/TestExpectations:
* LayoutTests/platform/mac/imported/w3c/web-platform-tests/css/css-color/system-color-compute-expected.txt: Copied from LayoutTests/imported/w3c/web-platform-tests/css/css-color/system-color-compute-expected.txt.
* LayoutTests/platform/mac/tables/mozilla/bugs/bug103533-expected.txt:
* Source/WebCore/rendering/RenderTheme.cpp:
(WebCore::RenderTheme::systemColor const):
* Source/WebCore/rendering/RenderThemeAdwaita.cpp:
(WebCore::RenderThemeAdwaita::systemColor const):
* Source/WebCore/rendering/RenderThemeIOS.mm:
(WebCore::RenderThemeIOS::systemColor const):
* Source/WebCore/rendering/RenderThemeMac.mm:
(WebCore::RenderThemeMac::systemColor const):

Originally-landed-as: 268124@main (a396d027d051). https://bugs.webkit.org/show_bug.cgi?id=245609
Canonical link: https://commits.webkit.org/269740@main
